sydent/http/servlets/__init__.py

```diff
@@ -228,13 +234,13 @@ def inner(*args, **kwargs):
     return inner
 
 
-def send_cors(request):
+def send_cors(request: Request) -> None:
     request.setHeader("Access-Control-Allow-Origin", "*")
     request.setHeader("Access-Control-Allow-Methods", "GET, POST, PUT, DELETE, OPTIONS")
     request.setHeader("Access-Control-Allow-Headers", "*")
 
 
-def dict_to_json_bytes(content):
+def dict_to_json_bytes(content: Dict[Any, Any]) -> bytes:
```

Comment:
JSON only allows string keys, so we can make this a bit more specific.

```suggestion
def dict_to_json_bytes(content: JsonDict) -> bytes:
```

(You'll need to import that at the top of the file!)